release-please-config.json

```diff
@@ -50,28 +50,28 @@
       "draft": false,
       "prerelease": false
     },
-    "packages/query": {
+    "packages/rpc": {
```

Comment:
Just renamed and relocated this one from `package/types` to `packages/rpc` ...others just shifted bc of the relocation.